validator.php

```diff
@@ -2,7 +2,11 @@
 
 // validates that all security advisories are valid
 
-require __DIR__.'/vendor/autoload.php';
+if (!is_file($autoloader = __DIR__.'/vendor/autoload.php')) {
```

Comment:
This... is kind of a normal thing in a composer project.

Feels like you're fighting the wrong error to me, or completely misunderstood what kind of project we're talking about...